src/components/toolbar/helpers.ts

```diff
@@ -85,6 +102,28 @@ const replaceDeepestSubsection = (
   )
 }
 
+const getDataTracked = (node: Node) => {
+  const dataTracked = node.attrs.dataTracked as TrackedAttrs[] | null
+  const change = dataTracked?.find((c) => c.operation === 'insert')
+  return change ? [change] : null
+}
+
+/** This to not include convert change in a deleted section of move */
```

Comment:
I didn't do that in my PR, I instead let the accept/reject handle such case.
Please update the name to indicate you are looking for the position of a previous **valid** node.

Reply:
`so you allow content to be moved to a deleted node` - No, this is not allowed. (https://github.com/Atypon-OpenSource/manuscripts-body-editor/blob/LEAN-4581/src/components/toolbar/helpers.ts#L320.)

Seems we are talking about 2 different things. Never mind!